solver-benchmarks/solver-benchmarks.cabal

```diff
@@ -41,6 +46,11 @@ library
     vector
   default-language: Haskell2010
 
+  if flag(Cabal-syntax)
```

Comment:
this package may depend on `Cabal & Cabal-syntax ==3.7.0.0` only, as its a test package

Reply:
Updated to only `Cabal-syntax >= 3.7`